homeassistant/components/fujitsu_hvac/config_flow.py

```diff
@@ -0,0 +1,79 @@
+"""Config flow for Fujitsu HVAC (based on Ayla IOT) integration."""
+from asyncio import timeout
+import logging
+from typing import Any
+
+from ayla_iot_unofficial import AylaAuthError, new_ayla_api
+from ayla_iot_unofficial.fujitsu_hvac import FujitsuHVAC
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import API_TIMEOUT, AYLA_APP_ID, AYLA_APP_SECRET, CONF_EUROPE, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(CONF_EUROPE): bool,
+    }
+)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Fujitsu HVAC (based on Ayla IOT)."""
+
+    def __init__(self) -> None:
+        """Create empty data."""
+        self.devices: dict[str, FujitsuHVAC] = {}
+        self.credentials: dict[str, str] = {}
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        errors: dict[str, str] = {}
+        if user_input is not None:
+            await self.async_set_unique_id(f"{user_input[CONF_USERNAME].lower()}")
```

Comment:
Can someone change the username?

Reply:
No : your username is your email address. You cannot change it once your account has been created.